cartographer/mapping_2d/pose_graph/constraint_builder.cc

```diff
@@ -246,6 +265,7 @@ void ConstraintBuilder::FinishComputation(const int computation_index) {
   std::unique_ptr<std::function<void(const Result&)>> callback;
   {
     common::MutexLocker locker(&mutex_);
+    kQueueLengthMetric->Set(constraints_.size());
```

Comment:
constraints_ is cleared further below in this function?

Reply:
Good point. I moved the measurement so we also measure this case.